google/cloud/security/iam/client.py

```diff
@@ -145,13 +147,30 @@ def query_access_by_resources(self, resource_name, permission_names,
 
     @require_model
     def query_access_by_members(self, member_name, permission_names,
-                                expand_resources=False):
+                                expand_resources=False,
```

Comment:
I know this method is not touched here first, but can you please document these method parameters and the return? These help with code quality, especially with python since they document the types.  We have agreed to maintain the method docs going forward.  Thanks!

You can see some examples here:
http://www.sphinx-doc.org/en/stable/ext/example_google.html